    
consept_mls0: 

  present_speed_setting: 1 # percent speed %
  L: 0.255 # khoang cach 2 banh
  Lm: 0.315 # Khoang cach  tu tam truc banh den tam do line 
  R: 0.075 # wheel radius (in meters per radian)
  Lt: 0.45 
  v_max_wheel: 100 # speed maximum of moter behind gear
  v_min_wheel: 2 # speed maximum of moter behind gear




consept_mls1: 

  present_speed_setting: 1 # percent speed %
  L: 0.255 # khoang cach 2 banh
  Lm: 0.355 # Khoang cach  tu tam truc banh den tam do line 
  R: 0.075 # wheel radius (in meters per radian)
  Lt: 0.45 
  v_max_wheel: 100 # speed maximum of moter behind gear
  v_min_wheel: 2 # speed maximum of moter behind gear


   